docs/reference/administration_reference.md

```diff
@@ -0,0 +1,82 @@
+# Administration reference guide
+
+## Overview
+
+This document provides a reference guide for all the attributes available for `invenio-administration` views.
+
+## List View
+
+### Attributes
+| Attribute                     | Required   | Default                                | Description                                                                                                    |
+|-------------------------------|------------|----------------------------------------|----------------------------------------------------------------------------------------------------------------|
+| **name**                      |            | *View class name*                      | - view's name used by Flask to create routing                                                                  |
+| **resource_config**           | *required* |                                        | used to retrieve the resource (refers to extension attribute - check ext.py)                                   |
+| **search_request_headers**    |            | *{ "Accept" :  "application/json" }    | specifies resource-specific headers to add during REST API request                                             |
+| **title**                     | *required* |                                        | displays a title in the list view page                                                                         |
+| **category**                  | *required* |                                        | places the menu entry under a category in the menu bar                                                         |                       
```

Comment:
Category is not required, the menu item will appear outside any category, as the Dashboard menu item.